src/addons/dragAndDrop/backgroundWrapper.js

```diff
@@ -72,6 +72,20 @@ class DraggableBackgroundWrapper extends React.Component {
   //   }
   // };
 
+  componentWillReceiveProps(nextProps) {
+    const { isOver: wasOver} = this.props;
+    const { isOver } = nextProps;
+    if (isOver && !wasOver) {
+      const { onEventResize, dragDropManager } = this.context;
+      const { value } = this.props;
+      const monitor = dragDropManager.getMonitor()
+      if (monitor.getItemType() === 'resize') {
+        // This was causing me performance issues so I commented it out. Thoughts? - Adam Recvlohe
+        // onEventResize('drag', {event: monitor.getItem(), end: value});
```

Comment:
what's going on here?

Reply:
I honestly don't know. This was carried over I think from what @jnmandal was doing I think.

Reply:
is think broken? If it's not needed lets remove it, otherwise don't we need to fix this?

Reply:
Ping

Reply:
Removed